lib/net.c

```diff
@@ -720,10 +720,62 @@ get_if_back(struct net_config *net_conf)
 	ETH_RSS_NONFRAG_IPV6_OTHER | \
 	ETH_RSS_IPV6_EX)
 
+static int
+check_port_rss_key_update(struct gatekeeper_if *iface, uint16_t port_id,
+	bool *rss_key_updatable, struct rte_eth_rss_conf *rss_conf)
```

Comment:
Add `const` to parameter `rss_conf` to guarantee that each port will see the same thing.